gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -901,14 +904,39 @@ def main(args):
         default=0.089,
     )
     compute_constraint_args.add_argument(
-        "--raw-z-outlier-threshold",
-        help=(
-            "Value at which the raw z-score is considered an outlier. Values below the"
-            " negative of '--raw-z-outlier-threshold' will be considered outliers for"
-            " lof and missense varaint counts (indicating too many variants), whereas"
-            " values either above '--raw-z-outlier-threshold' or below the negative of"
-            " '--raw-z-outlier-threshold' will be considered outliers for synonymous"
-            " varaint counts (indicating too few or too many variants)."
+        "--raw-z-outlier-threshold-lof",
```

Comment:
Should we have these also have lower in the name just to be very obvious?

Reply:
will add